tests/kernel/fp_sharing/testcase.yaml

```diff
@@ -1,7 +1,7 @@
 tests:
   kernel.fp_sharing:
     extra_args: PI_NUM_ITERATIONS=70000
-    platform_whitelist: frdm_k64f
+    platform_whitelist: frdm_k64f sam_e70_xplained
```

Comment:
I wondering if this should be based on a CONFIG option instead of explicit platform choice.